Corona-Warn-App/src/main/res/layout/tracing_details_item_periodlogged_view.xml

```diff
@@ -60,46 +60,5 @@
                 app:layout_constraintTop_toBottomOf="@id/risk_details_period_logged_subtitle" />
         </androidx.constraintlayout.widget.ConstraintLayout>
 
-        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
After removal, it looks like this, the text doesn't make sense anymore, we need to adjust it.

(Increased/Low Risk State)

![Screenshot from 2021-03-04 13-04-42](https://user-images.githubusercontent.com/1439229/109961444-49f64580-7cea-11eb-803b-75552209beb8.png)

Reply:
Hm 

cc/ @janetback

Reply:
@harambasicluka @ralfgehrer or do we replace this with a new text/number from the day counter? EXPOSUREAPP-5170 ?

Reply:
I think this should be covered by my ticket 5170.